### ISSUE TITLE
[SCREENREADER]: 'Directions' links SHOULD have accessible names

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#6894]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Each of the "Directions" links in the search results should have an accessible name, which distinguishes it from the other links. Consider adding text to give each unique accessible names, for those who are blind or deafblind. This might be done dynamically, and could be added for screen readers only.

![20200316-facilities-accessible-names](https://user-images.githubusercontent.com/57469/76771424-b42e7700-6775-11ea-829f-d3914f7f9ccd.png)

Here's is a visual representation of what a screen reader user hears when they navigate by links:
![IMG_6699](https://user-images.githubusercontent.com/57469/77690442-a9e05a00-6f79-11ea-8eb9-03f5bd3086a1.jpg)

### Why it Matters

This is important because the intention is to to help users understand the purpose of each link in the content, so they can decide whether they want to follow it. Best practice is that links with the same destination would have the same descriptions, but links with different purposes and destinations would have different descriptions (see also [Success Criterion 3.2.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-consistent-functionality.html) which calls for consistency in identifying components that have the same functionality). Because the purpose of a link can be identified from its link text, links can be understood when they are out of context, such as when the user agent provides a list of all the links on a page.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

- As a screen reader user, I want to know with certainty what links will do if I activate them.
- As a screen reader user navigating a page by links, I want to be able to distinguish what each link will do, and may not have the surrounding context.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: production

## Steps to Recreate

1. Enter `https://www.va.gov/find-locations` in browser
2. Start screenreading device of your choice
3. Navigate the page by tabbing (which will tab through the interactive elements)
4. Notice that the directions links are only read as 'link directions'; it's good that the title of the location is also a link which is ready juste before the directions link, so this may not be a big issue.

## Possible Fixes (optional)

```diff
<!-- current code -->
<a href="https://maps.google.com?saddr=Current+Location&amp;daddr=128 Southwest Chamber Court, Port Saint Lucie, FL 34986-3496" rel="noopener noreferrer" target="_blank" style="text-decoration: underline;">
!    Directions
</a>

<!-- possible fix -->
<a href="https://maps.google.com?saddr=Current+Location&amp;daddr=128 Southwest Chamber Court, Port Saint Lucie, FL 34986-3496" rel="noopener noreferrer" target="_blank" style="text-decoration: underline;">
+    Directions <span class="sr-only">to Port Saint Lucie <abbr title="V.A.">VA</abbr> Clinic</span
</a>
```

## WCAG or Vendor Guidance 

* [WCAG 2.4.9 Ensure that links with the same accessible name serve a similar purpose](https://dequeuniversity.com/rules/axe/3.5/identical-links-same-purpose?application=AxeChrome)

**Comment by mmiddaugh** [comment]
Pulling this one in as it was adjacent work to the work in [#10324](https://github.com/department-of-veterans-affairs/va.gov-team/issues/10324)

**Comment by mmiddaugh** [comment]
@jenstrickland this is ready to validate when you are able.

**Comment by jenstrickland** [comment]
❤️  Validated, this is great! Thanks!
